mautic/mautic
# Update/Fix sample emails for Mautic 5

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #13966 



## Description


The sample emails loaded by running `doctrine:fixtures:load` could not be opened because they were missing the field "custom_html". This PR fixes that issue.


---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. In the terminal, run `ddev exec php bin/console d:f:l` and answer "yes" to load sample data
3. Go to "Emails" and click on one of the Kaleidoscope emails.
4. Verify that you are taken to the details page
![image](https://github.com/user-attachments/assets/7e1f0791-d545-493f-b2ba-a1669b643555)

5. Click on Edit and then on Builder
6. Verify that you can see the email's content
![image](https://github.com/user-attachments/assets/1beba557-d4d1-49ca-9808-171e4915ee88)
